MdeModulePkg/Universal/Acpi/AcpiTableDxe/AcpiTableProtocol.c

```diff
@@ -552,13 +557,17 @@ AddTableToList (
     // could be updated by OS present agent. For example, BufferPtrAddress in
     // SMM communication ACPI table.
     //
-    ASSERT ((EFI_PAGE_SIZE % 64) == 0);
-    Status = gBS->AllocatePages (
-                    AllocateMaxAddress,
-                    EfiACPIMemoryNVS,
-                    EFI_SIZE_TO_PAGES (CurrentTableList->TableSize),
-                    &AllocPhysAddress
-                    );
+    if (IsFromHob){
+      AllocPhysAddress = (UINTN)Table;
+    } else {
+      ASSERT ((EFI_PAGE_SIZE % 64) == 0);

```

Comment:
Is this required  ?   EFI_PAGE_SIZE is already page aligned.

Reply:
Thanks for catching this.
I agree it is not needed, and will remove it